buildpacks/ruby/src/layers/metrics_agent_install.rs

```diff
@@ -0,0 +1,286 @@
+use crate::build_output;
+use crate::{RubyBuildpack, RubyBuildpackError};
+use flate2::read::GzDecoder;
+use libcnb::data::layer_content_metadata::LayerTypes;
+use libcnb::layer::ExistingLayerStrategy;
+use libcnb::{
+    additional_buildpack_binary_path,
+    generic::GenericMetadata,
+    layer::{Layer, LayerResultBuilder},
+};
+use serde::{Deserialize, Serialize};
+use std::os::unix::fs::PermissionsExt;
+use std::path::{Path, PathBuf};
+use tar::Archive;
+use tempfile::NamedTempFile;
+
+/// Agentmon URL
+///
+/// - Repo: <https://github.com/heroku/agentmon>
+/// - Releases: <https://github.com/heroku/agentmon/releases>
+///
+/// To get the latest s3 url:
+///
+/// ```shell
+/// $ curl https://agentmon-releases.s3.amazonaws.com/latest
+/// ```
+const DOWNLOAD_URL: &str =
+    "https://agentmon-releases.s3.amazonaws.com/agentmon-0.3.1-linux-amd64.tar.gz";
```

Comment:
```suggestion
/// $ curl https://agentmon-releases.s3.us-east-1.amazonaws.com/latest
/// ```
const DOWNLOAD_URL: &str =
    "https://agentmon-releases.s3.us-east-1.amazonaws.com/agentmon-0.3.1-linux-amd64.tar.gz";
```

(For context, see https://github.com/heroku/agentmon/pull/18)